Doc/requirements-oldest-sphinx.txt

```diff
@@ -36,3 +36,4 @@ sphinxcontrib-jsmath==1.0.1
 sphinxcontrib-qthelp==1.0.3
 sphinxcontrib-serializinghtml==1.1.5
 urllib3==1.26.15
+sphinx-copybutton<0.6
```

Comment:
This file is meant to be a generated list of pinned dependencies, see instructions at the top.